app/views/search/foods.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Food Result Page</h1>
+
+<h3>Results:<h3>
+
+Results Count: <%= @foods.count %><br>
```

Comment:
There is an even easier way to get this data if you check out the response from that USDA endpoint